app/AccountancyModule/PaymentModule/presenters/AddRegistrationMembersPresenter.php

```diff
@@ -71,13 +71,18 @@ public function actionDefault(int $id, ?int $unitId = null): void
         $list = array_slice($list, 0, 50);
 
         foreach ($list as $p) {
+            $totalAmount = (float) $p['AmountTotal'];
+            if ($totalAmount < 1) {
```

Comment:
Nebylo by tady lepší kontrolovat, že to není nula? I když je to asi nepravděpodobné, tak i např 0,50 je pořád validní částka, kterou to teď zahodí.

Reply:
já jsem se chtěl vyhnout zaokrouhlací chybě, ale muzu to upravit na `>0`

Reply:
To pak případně není problém řešit skrze použití Money VO, který zaokrouhlovací chyby eliminuje.